src/mixxx.cpp

```diff
@@ -735,24 +733,30 @@ void MixxxMainWindow::finalize() {
     // https://bugs.launchpad.net/mixxx/+bug/1882474
     // https://bugs.launchpad.net/mixxx/+bug/1909485
     // So let's quit fullscreen if StartInFullscreen is not checked in Preferences.
-    bool fullscreenPref = m_pSettingsManager->settings()->getValue<bool>(
-            ConfigKey("[Config]", "StartInFullscreen"));
-    if (isFullScreen() && !fullscreenPref) {
-        slotViewFullScreen(false);
-        // After returning from fullscreen the main window incl. window decoration
-        // may be too large for the screen.
-        // Maximize the window so we can store a geometry that fits the screen.
-        showMaximized();
-    }
-    m_pSettingsManager->settings()->set(ConfigKey("[MainWindow]", "geometry"),
-        QString(saveGeometry().toBase64()));
-    m_pSettingsManager->settings()->set(ConfigKey("[MainWindow]", "state"),
-        QString(saveState().toBase64()));
+    UserSettingsPointer pSettings;
+    if (m_pSettingsManager) {
```

Comment:
Und which circumstances can Settings Manager or Settings be null?

Reply:
Ah I see due to later commits, it can no longer be null. I will revert that.

Reply:
I hope it is OK to just drop the commit when rebasing.